docs/source/tutorials/building_blocks.rst

```diff
@@ -22,15 +22,15 @@ Mobjects
 
 Mobjects are the basic building block for all manim animations.  Each class
 that derives from :class:`.Mobject` represents an object that can be displayed
-on screen.  For example, simple shapes such as :class:`.Circle`,
+on the screen.  For example, simple shapes such as :class:`.Circle`,
 :class:`.Arrow`, and :class:`.Rectangle` are all mobjects.  More complicated
 constructs such as :class:`.Axes`, :class:`.FunctionGraph`, or
 :class:`.BarChart` are mobjects as well.
 
-If you try to display on screen an instance of :class:`.Mobject`, you will only
+If you try to display on the screen an instance of :class:`.Mobject`, you will only
```

Comment:
original sounds better here.

Reply:
Again it’s based on your preference.

Reply:
this time I'm with @Nikhil0504 but we could also reformulate further:

If you try to display an instance of mobject on the screen, you will only